src/pages/user_profile/UserProfile.js

```diff
@@ -124,12 +146,25 @@ export default function UserProfile(props) {
     >
       Full Name
     </h1>
-    <FormField
-      type={FormFieldTypes.TEXT}
-      id='profileName'
-      defaultValue={profile.profileName}
-      disabled={true}
-    />
+    <div style={{ display: 'flex', alignItems: 'center' }}>
+      <FormField
+        type={FormFieldTypes.TEXT}
+        id='profileName'
+        defaultValue={name}
+        onChange={updateRef}
+        style={{ width: '90%', marginTop: '10px', 
```

Comment:
nit: extra space at end of line
```suggestion
        style={{ width: '90%', marginTop: '10px',
```